ui/src/components/Charts/BarChart.js

```diff
@@ -141,7 +143,7 @@ class BarChart extends Component {
         <div className={`${styles.withPrevNext} clearfix`} />
         <div className={`${styles.barContainer}`}>
           <span className={styles.yAxis}>
-            {this.props.model.get('axesyLabel') || this.props.model.getIn(['axesgroup', 'searchString'], 'Y-Axis')}
+            {this.props.model.get('axesyLabel') || shorten(this.props.model.getIn(['axesgroup', 'searchString'], 'Y-Axis'))}
```

Comment:
Just check @Ian247 is okay with this because I believe this is what we previously had and everyone hated it.

Reply:
Decided it's fine because users are able to write their own labels to override it.